src/domain/targeted-messaging/targeted-messaging.repository.ts

```diff
@@ -23,6 +25,12 @@ export class TargetedMessagingRepository
     return this.datasource.getTargetedSafe(args);
   }
 
+  async createTargetedSafes(
+    args: CreateTargetedSafesDto,
+  ): Promise<TargetedSafe[]> {
```

Comment:
Nit: The array types in CGW are usually defined like:
`): Promise<Array<TargetedSafe>> {`

Reply:
https://github.com/safe-global/safe-client-gateway/commit/1b72d7acb028af5275397cac1a6b18e63e85e435#diff-9bc05b83e026841bc7b3f0dd8c8b4ace8e81ca26aacdf3d56079dd5215316b22L20